stdlib/src/builtin/value.mojo

```diff
@@ -121,3 +121,15 @@ trait StringableCollectionElement(CollectionElement, Stringable):
     """
 
     pass
+
+
+trait ComparableCollectionElement(CollectionElement, EqualityComparable):
```

Comment:
**Suggestion** Do you mind rebasing with the latest `nightly` from today's release since this trait exists now at https://github.com/modularml/mojo/blob/ee5218693b57959d44e9b82dbc0774fb4d28db18/stdlib/src/builtin/value.mojo#L126?

Reply:
I rebased but now it has added too many files🥲

Reply:
Done